spec/features/job_instance_spec.rb

```diff
@@ -25,10 +25,20 @@
 
     expect(page).to have_selector('#launchAdHocModal .modal-dialog', visible: true)
     within '#launchAdHocModal .modal-dialog' do
+      # retry 5 times to wait appearing Lauch button
+      5.times do
+        break if find_button('Launch').visible?
+        sleep 1
+      end
+
       click_on 'Launch'
```

Comment:
Perhaps, the reason why this spec is failed is not to failed to move next page, but not to put the `Lauch` button at L34

Reply:
🏆  👍